presto-jdbc/src/test/java/io/prestosql/jdbc/TestPrestoDriver.java

```diff
@@ -1435,7 +1435,7 @@ public void testQueryCancelByInterrupt()
         AtomicReference<Throwable> queryFailure = new AtomicReference<>();
 
         Future<?> queryFuture = executorService.submit(() -> {
-            try (Connection connection = createConnection("blackhole", "default");
+            try (Connection connection = createConnection("blackhole", "blackhole");
```

Comment:
Are the changes in this file necessary?

Reply:
@findepi  They are :) . Since before the schema name is not used to lookup the table, so it doesn't matter which schema you used... you still end up find the table in other schemas :)

Reply:
@wenleix  thanks for the clarification!